### DISCUSSION TITLE
[WP2261 Discussion]: ERK - c-myc relationship is needed

### DISCUSSION THREAD

**Discussion by tturboscience** [wikipathways/wikipathways-help#101] [pathway-discussion]
### Discussion

This curated chart shows the most commonly mutated genes in glioblastomas, based on a 2008 publication. However, not least important is the c-myc gene, known as the emperor of oncogenes: "High-level amplification of the MYC and MYCN genes are observed in a subset of GBM (7-11). Myc is therefore a compelling therapeutic target in GBM." (from Tateishi K, Iafrate AJ, Ho Q, Curry WT, Batchelor TT, Flaherty KT, Onozato ML, Lelic N, Sundaram S, Cahill DP, Chi AS, Wakimoto H. Myc-Driven Glycolysis Is a Therapeutic Target in Glioblastoma. Clin Cancer Res. 2016 Sep 1;22(17):4452-65. doi: 10.1158/1078-0432.CCR-15-2274. Epub 2016 Apr 13. PMID: 27076630; PMCID: PMC5010492.).
It is therefore worth adding the ERK-c-myc relationship

Translated with DeepL.com (free version)

### Type of revision

- [ ] Major
- [X] Minor
- [ ] Deletion

**Comment by khanspers** [comment]
Hi @tturboscience, thank you for the suggestion! Please feel free to add MYC and MYCN to the pathway, with the relevant literature reference.

**Reply by khanspers** [reply]
To open any pathway from WikiPathways in PathVisio (using the WikiPathways plugin), search for the pathway by WPID under Plugins>WikiPathways>Search. After making edits, select Plugins>WikiPathways>Update to submit the updated pathway back to WikiPathways. You will be prompted to sign in using your WikiPathways account.

**Comment by tturboscience** [comment]
Thank you, Kristina!

Of course, one could add some connection of c-myc to glioma and other
pathways,
but first of all I see a common problem for all cancer pathways:
K-ras, c-myc, mTor1 and tp53 genes working together in the mainstream of
carcinogenesis and cell malignancy. In recent years I have been exploring
these links and intend to propose a model.
It will take some time to study the vast amount of fragmentary papers on
this topic. Let other researchers consider this important issue.

Regards - Kest

On Fri, 2 Feb 2024 at 21:28, Kristina Hanspers ***@***.***>
wrote:

> To open any pathway from WikiPathways in PathVisio (using the WikiPathways
> plugin), search for the pathway by WPID under Plugins>WikiPathways>Search.
> After making edits, select Plugins>WikiPathways>Update to submit the
> updated pathway back to WikiPathways. You will be prompted to sign in using
> your WikiPathways account.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/wikipathways/wikipathways-help/discussions/101#discussioncomment-8350149>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMVGUITCVSYPB7BGSAD76QTYRU45BAVCNFSM6AAAAABCUVLLCWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGNJQGE2DS>
> .
> You are receiving this because you were mentioned.Message ID:
> <wikipathways/wikipathways-help/repo-discussions/101/comments/8350149@
> github.com>
>


-- 
Pagarbiai - Kęstutis

**Comment by Chris-Evelo** [comment]
Hi Kest,

That sounds *very* interesting. It is more the relation between these regulatory genes than the individual genes that are important. I am also in the ELIXIR cancer data community, where people might have an interest in what you are trying to do. Maybe we can share some of your initial work, when done, with them and ask them to participate?

Best, Chris

**Comment by tturboscience** [comment]
Thank you. My work is contained in the manuscript of the monograph "The K-ras problem in oncology: a systems approach", which is based on more than a thousand individual scientific papers and is more than 250 pages long - it summarises 35 years of theoretical work in the field of systems research since I won the 1st Lithuanian Young Scientists Competition. I am a theoretician looking for an answer to the question - whether the more than 5 million scientific publications and the approximately 100,000 clinical trials carried out, which in many cases scare me because of their poor logical validity, are already able to produce coherent scenarios of carcinogenesis-malignancy - or whether there is still something more that needs to be investigated.  At the moment I'm finalizing the WikiPathway chart, which describes 41 the hallmark, feature and process descriptor of  cancer at biology level.  I will then try to shed some light on the molecular level processes in relation to cancer biological descriptors

**Reply by Chris-Evelo** [reply]
Sounds great! If you want I can ask the Cancer Data community whether they are interested and you could maybe give a short introduction there?

**Reply by tturboscience** [reply]
Dear Chris, ELIXIR - Cancer data... is working on organising and sharing empirical cancer surveillance data in order to produce science-based conclusions. This is philosophically an empirical approach based on inductive logic. My methodology is about bringing together fragmented scientific knowledge into a whole - into as complete and coherent a mosaic of models as possible. Philosophically, this is an application of deductive logic: the data I am currently working with are not from experiments, but only individual facts of scientific knowledge that are highly relevant, yet fragmented in their thinking. There is an ocean of empirical data and a huge dearth of organised knowledge: I count over twenty theories and concepts of carcinogenesis alone.  This is the essence of my theoretical work, helped by the work of WikiPathways, despite the large fragmentation. The application of AI in deduction from the field of "text mining" may provide a solution. I don't know what communities in the cancer field are doing this

**Comment by Chris-Evelo** [comment]
Thanks! OK then I will do it the other way around and ask the cancer data community whether they know where this is happening.

**Comment by tturboscience** [comment]
Well, It occurred to me how it might be possible to use the ELIXIR-cancer community to efficiently develop models of carcinogenesis-malignancy. This would require extracting statistics from the accumulated data on how often changes occur in essential and key genes in cancer cells: k-ras, c-myc, mTor1 (which I call the "Bermuda Triangle") and tp53. Of course, there are many different options, probably each one of them can be an initiator, not only a driver, so the task still needs to be very well thought out, but first, as I said, the model needs to be completed, and before that we need to organize the cancer description in those 41 biological level descriptions and make it available in the form of WikiPathways. This takes some time, but the task has already taken shape - thank you

**Comment by tturboscience** [comment]
Some technical problems appeared: unfortunately I can't upload the new recently constructed WikiPathway having three levels of description for Pancreatic cancer :

"Developing a systems approach as opposed to a fragmented one, the description of pancreatic cancer is presented at three levels:
 - Clinical oncology language and classification, 
- biological cancer hallmarks, features and processes
- molecular and elementary particle processes.

The WHO cancer classification, the classic Hanahan-Weinberg model of cancer biology (2011) with an extension (2022) and the ontology of cancer biology in an elaborating manner (S.Baker et al. 2017) are used, which is linked to the corresponding WikiPathways developed by the PancCanNet Community (in the scheme WP...) and by the other authors  (in the scheme lowercase wp...).  Several missing WP pathways have been replaced by the reference of the scientific article (in the scheme triangle). Kęstutis K.Urba"
